packages/runtime/container-runtime/src/test/dataStoreCreation.spec.ts

```diff
@@ -114,7 +114,7 @@ describe("Data Store Creation Tests", () => {
 				on: (event, listener) => {},
 				logger: createChildLogger(),
 				clientDetails: {},
-			} as ContainerRuntime;
+			} as unknown as ContainerRuntime;
```

Comment:
I think this cast shouldn't be needed if you update the object to have a `baseLogger` property.

Reply:
Hmmm...It was complaining if I do "as ContainerRuntime" I need to set 148 parameters.